packages/flutter/lib/src/cupertino/localizations.dart

```diff
@@ -208,16 +208,24 @@ class _CupertinoLocalizationsDelegate extends LocalizationsDelegate<CupertinoLoc
   const _CupertinoLocalizationsDelegate();
 
   @override
-  bool isSupported(Locale locale) => locale.languageCode == 'en';
+  bool isSupported(Locale locale) => <String>['en', 'fr'].contains(locale.languageCode);
```

Comment:
Let's make this a set -> `<String>{'en, 'fr'}.contains(local.languageCode);`

Reply:
Actually, it'd probably be good to comment these so that as it grows we know, e.g. 
```
const Set<String> kSupportedLanguages = <String>{
  'en', // English
  'fr', // French
};

@override
bool isSupported(Locale locale) => kSupportedLanguages.contains(locale.langaugeCode);
```

Reply:
I thank you for your feedback. I made the modification.

Sorry for the delay